### PR TITLE
Auto-scroll to current step in sequence table

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**

### CODE REVIEW

File: modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala [hunk 1]
```diff
@@ -128,6 +129,12 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
     extends SequenceTableDefs[D]:
   private type Props = SequenceTable[S, D]
 
+  private def scrollIfNeeded(element: dom.Element) = Callback {
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala:132]: Do we have this function in explore too? maybe eventually move it to lucuma-ui

Comment (hugo-vrijswijk) [reply modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala:132]: Yeah we have it in one place

**Review by cquiroz — APPROVED** [review]
LGTM

**Comment by rpiaggio** [comment]
I think we may need to involve the virtualizer in this, like we do in the configuration table in explore. The step may not be rendered if it's beyond the current rows shown by the virtualizer.

Also, it may be nice to scroll to the next unexecuted step even if it's not selected.

**Pull request merged**